cluster/image.go

```diff
@@ -126,9 +127,33 @@ func (images Images) Filter(opts ImageFilterOptions) Images {
 		return false
 	}
 
+	danglingFilter := func(image *Image) bool {
+		if opts.Filters.Include("dangling") {
+			if len(image.RepoTags) == 0 {
+				image.RepoTags = []string{"<none>:<none>"}
```

Comment:
The `Filter` function is to find images that matches filters. It should be read-only, i.e., not changing another data in images.